docs/section19/19-23.md

```diff
@@ -0,0 +1,77 @@
+# Section 19 - Rule 19-23           
+**Schema Version:** 0.0.23    
+**Mandatory Rule:** True    
+**Rule ID:** 19-23         
+**Rule Description:** The calculated system fan power shall be distributed to supply, return, exhaust, and relief fans in the same proportion as the proposed design.   
+
+**Rule Assertion:** Options are Pass/Fail    
+**Appendix G Section:** G3.1.2.9          
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:** None   
+
+**Function Calls:**  
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()    
+2. get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM() 
+3. get_zone_supply_return_exhaust_relief_terminal_fan_power_dict()  
+4. match_data_element()  
+5. get_component_by_id()  
+
```

Comment:
Fan power of the baseline VAV fan-powered boxes in systems 6 & 8 is prescribed and I think should not participate in the "allocation". Seems like the logic below does not exclude them since line 43 looks at all fans at zone level. I wonder then if we should also exclude zonal fans in the proposed design from the target percent allocation to have apples to apples comparision?

Reply:
good point. we need to include terminal fans as FPFCU use that nomenclature. To exclude PIUs, we might need to check afn configuration and exclude when it is series or parallel?

Reply:
@mkarpman @supriyagoel Shouldn't we exclude terminal fans in the proposed since they have a separate fan power allocation in the baseline?

Reply:
OK, I realized that my original comment wasn't accurate as it was referring to fans at the zone level. So to correct myself, consider scenario when both the baseline and proposed designs have PVAV system. In that case, the total fan energy associated with a zone would include the prorated energy of the FanSystem of the associated HVAC zone level, and also energy from Terminal.Fan that would represent the fan in the VAV box.

Reply:
@mkarpman OK, so you think the VAV terminal fans should be included for apportioning the baseline fan power allowance? This does not make sense to me since there is a separate fan power allowance for the VAV fan powered boxes in the baseline. How would that terminal fan power from the proposed be apportioned in the baseline then since the allowance is set? Perhaps the two of us should speak about this RDS outside of the group call to get on the same page.

Reply:
We are on the same page. Resolved.